modules/client-configuration/pages/clients-ubuntu.adoc

```diff
@@ -165,13 +171,11 @@ endif::[]
 ifeval::[{suma-content} == true]
 .Procedure: Alternative Method for Adding {ubuntu} Channels and Repositories
 . At the command prompt on the {productname} Server, as root, install the [systemitem]``spacewalk-utils`` package:
-+
```

Comment:
I removed them, at the generated HTML doc the `+` was shown, and I guess that is not wanted.

Reply:
Removing them will mean the code block pops out of the step block, though, and then renumbering will start again. All that said, they shouldn't be showing either :woman_shrugging:

Reply:
Let's merge, and maybe you can have a look on your branch :-)